section-3/exercises/vsnitsar-section3.js

```diff
@@ -0,0 +1,173 @@
+console.log(latestCar);
+
+/*
+Exercise 3:
+Create an array cars with "Toyota", "Ford", "BMW".
+1. Add "Tesla" to the end of array.
+2. Remove "Toyota" car from cars array and print the array.
+3. Check if array includes "Ford" in cars array
+4. Add Mercedes as a 1st element of cars array
+*/
+
+console.log("Exercise 3.1.")
+const cars = ['Toyota', 'Ford', 'BMW'];
+cars.push('Tesla');
+
+console.log("Exercise 3.2.")
+cars.splice(0, 1);
+console.log(cars);
+
+console.log("Exercise 3.3.")
+console.log("Cars array includes 'Ford': ", cars.includes('Ford'));
+
+console.log("Exercise 3.4.")
+cars.unshift('Mercedes');
+
+/*
+Exercise 4:
+What will show console log in the code bellow ? Think without code execution
+    const cars = ["Toyota", "Ford", "BMW", 'Tesla'];
+    const carsTwo = cars;
+    carsTwo.push('Mercedes');
+    console.log(cars.length);
+*/
+
+//Answer: both will be changed because of reference type of array, and both will have length 5
+
+/*
+Exercise 5:
+Create an object calculator with properties num1 and num2, add method add() that returns their sum.
+*/
+
+const calculator = {
+    num1: undefined,
+    num2: undefined,
+    add: function (num1, num2) {
+        return num1 + num2;
+    },
+}
+console.log('Sum of two values = ', calculator.add(1, 2));
+
+/*
+Exercise 6:
+You have an array of 12 months
+    const twelveMonths = [ 'January', 'February', 'March', 'April', 'May', 'June', 'July',
+    'August', 'September', 'October', 'November', 'December'];
+From twelveMonths array extract months which are related to autumn in separate array.
+*/
+console.log("Exercise 6.")
+const twelveMonths = ['January', 'February', 'March', 'April', 'May', 'June', 'July',
+    'August', 'September', 'October', 'November', 'December'];
+console.log("Method 1:");
+const autumnMonths = twelveMonths.slice(8, 11);
+console.log(autumnMonths);
+console.log("Method 2:");
+twelveMonths.splice(0, 8);
+twelveMonths.pop(); //or twelveMonths.splice(-1, 1), don't manage how to do it in 1 step
+console.log(twelveMonths);
+/*
+
+Exercise 7:
+You have two arrays
+    const fruits1 = ["Apple", "Banana"];
+    const fruits2 = ["Cherry", "Mango"];
+Merge these 2 arrays to one array using all methods which you know;
+*/
+console.log("Exercise 7.");
+const fruits1 = ["Apple", "Banana"];
+const fruits2 = ["Cherry", "Mango"];
+
+console.log("Method 1: ", fruits1.concat(fruits2));
+console.log("Method 2: ", [...fruits1, ...fruits2]);
+console.log("Method 3:");
+fruits1.push(...fruits2)
+console.log(fruits1);
+/*
+Exercise 8:
+1. Merge two objects { a: 1, b: 2 } and { c: 3, d: 4 } to one object;
+2. For the merged object print object keys & object values;
+*/
+firstObject = {
+    a: 1,
+    b: 2
+}
+secondObject = {
+    c: 3,
+    d: 4
+}
+newObject = Object.assign({}, firstObject, secondObject);
+console.log(newObject);
+
+console.log("Keys of new object are: " + Object.keys(newObject));
+console.log("Values of new object are: " + Object.values(newObject));
+
```

Comment:
You missed variable declaration in this exercises ( firstObject, secondObject, newObject). Add variable declaration please